### PR TITLE
feat: hello codespell

### PR DESCRIPTION
**Pull request by kesselb**
### ☑️ Resolves

* Fix #…

### CODE REVIEW

**Comment by hamza221** [comment]
Codespell is faling 😅

File: admin_manual/configuration_database/linux_database_configuration.rst [hunk 1]
```diff
@@ -208,7 +208,7 @@ like this:
   extension=pdo_pgsql.so
   extension=pgsql.so
 
-  [PostgresSQL]
+  [PostgreSQL]
```
Comment (nickvergessen) [inline admin_manual/configuration_database/linux_database_configuration.rst:211]: Cool the docs example is even broken, `[PostgreSQL]` is correct :see_no_evil:

Comment (kesselb) [reply admin_manual/configuration_database/linux_database_configuration.rst:211]: Yep. 

Codespell changed `PostgresSQL` to `Postgressql` and I then checked the value and corrected it.

File: .codespellignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SME
+te
+jus
```
Comment (kesselb) [inline .codespellignore:3]: image

:smile:

**Review by nickvergessen — APPROVED** [review]

File: admin_manual/configuration_database/linux_database_configuration.rst (hunk 1 above)
Comment (nickvergessen) [reply admin_manual/configuration_database/linux_database_configuration.rst:211]: Should backport this line at least. Or you want to backport codespell completely?

**Pull request merged**